src/components/FileSelector/styled/index.js

```diff
@@ -60,7 +61,7 @@ export const StyledDropzone = attachThemeAttrs(styled.div)`
             cursor: not-allowed;
             border-color: ${props.palette.text.disabled}
         }
-        `}
+    `}
 
     ${props =>
         props.error &&
```

Comment:
the `background-color` when `error` also should be transparent, we should use anything like that:

`background-color: ${replaceAlpha(props.palette.error.main, 0.1)};`